azext_iot/_params.py

```diff
@@ -72,8 +73,10 @@ def load_arguments(self, _):
         context.argument('auth_method', options_list=['--auth-method', '-am'],
                          arg_type=get_enum_type(DeviceAuthType),
                          help='The authorization type an entity is to be created with.')
-        context.argument('content', options_list=['--content', '-k'],
-                         help='IoT Edge configuration content json. Provide file path or raw json.')
+        context.argument('metric_type', options_list=['--metric-type', '-mt'], arg_type=get_enum_type(MetricType),
```

Comment:
do they still support -mt, or if we want to move away from this convension in the future?

Reply:
that breaking change was rolled back, but its a good idea to stay away from the convention for the future